modules/xdcr-reference/pages/xdcr-advanced-settings.adoc

```diff
@@ -37,6 +37,8 @@ This value must be less than or equal to that specified for `XDCR Target Nozzles
 A small value of 2 or 4 is often sufficient.
 The default is 2. The value-range is 1-100.
 
+Note that if the specified value is too high for the currently established number of xref:manage:manage-settings/general-settings.adoc#xdcr-maximum-processes[XDCR Maximum Processes], Couchbase Web Console displays a warning.
```

Comment:
I'm thinking it would be good to add the clause "in the XDCR Advanced Settings" so that people aren't thinking that the warning is displayed in the main XDCR window.

Example:
Note that if the specified value is too high for the currently established number of xref:manage:manage-settings/general-settings.adoc#xdcr-maximum-processes[XDCR Maximum Processes], Couchbase Web Console displays a warning in the XDCR Advanced Settings.